packages/ketcher-react/src/script/ui/views/components/Dialog/Dialog.module.less

```diff
@@ -0,0 +1,23 @@
+@import '../../../../../style/mixins.less';
+
+.close {
+  .reset-button;
```

Comment:
seems strange, if this class is redundant, please remove it

Reply:
@TomasLight, `.reset-button` is a mixin, it allows us to add additional styles to the `.close` class.